app/components/preprint-status-banner/style.scss

```diff
@@ -64,6 +64,9 @@
     .reviewer-feedback {
       padding-right: 15px;
 
+      i {
```

Comment:
Does this impact all of the other icons in the dropdown? We probably want to have a separate class for the loading indicator in the button.

Reply:
that is true..sorry will update